external/fv3fit/fv3fit/keras/_models/convolutional.py

```diff
@@ -125,7 +136,13 @@ def train_convolutional_model(
     train_Xy = get_Xy(data=train_batches, clip_config=hyperparameters.clip_config)
     # need unclipped shapes for build_model
     # have to batch data so we can take statistics for scaling transforms
-    X, y = next(iter(get_Xy(data=train_batches, clip_config=None).batch(10_000_000)))
+    X, y = next(
```

Comment:
Copied over normalization fitting logic from the dense training. Might want to consolidate these later into some `fit_nd_model` helper (instead of only `fit_column_model`).